### PR TITLE
Add release-v0.19 yaml

### PR DESCRIPTION
**Pull request by vdemeester**
Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

### CODE REVIEW

File: openshift/release/tektoncd-pipeline-v0.19.0.yaml [hunk 1]
```diff
@@ -0,0 +1,2070 @@
...
+        app: tekton-pipelines-controller
+        version: v0.19.0
+    spec:
+      serviceAccountName: tekton-pipelines-controller
+      containers:
+      - name: tekton-pipelines-controller
+        image: quay.io/openshift-pipeline/tektoncd-pipeline-controller:v0.19.0
+        args: [
+          # Version, to be replace at release time
+          "-version", "devel",
```
Comment (piyush-garg) [inline openshift/release/tektoncd-pipeline-v0.19.0.yaml:1655]: We need to fix the script for this

Comment (vdemeester) [reply openshift/release/tektoncd-pipeline-v0.19.0.yaml:1655]: Interesting, I don't get why it didn't replace this though.. looking at the script, it should have…

**Pull request merged**